src/main/java/au/org/democracydevelopers/raireservice/request/ContestRequest.java

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Do you want to copy the template I have used in the CVR database retrieval branch? It has some line breaks and conforms to the line width limit.